bitbots_vision/src/bitbots_vision/vision_modules/ros_utils.py

```diff
@@ -142,18 +143,26 @@ def add_model_enums(cfg_type, package_path):
                 'name': folder,
                 'value': folder,
                 'description': 'fcnn {}'.format(folder)})
-        # Is this model an yolo model
+        # Is this model a yolo darknet model
         elif os.path.exists(os.path.join(models_directory, folder, "yolo_weights.weights")):
             # Append list with a new enum item
-            yolo_paths.append({
+            yolo_darknet_paths.append({
+                'name': folder,
+                'value': folder,
```

Comment:
Why is there name and value with both having the same value?

Reply:
We need the two fields for rqt_dynamic_reconfigure, but the displayed name should equal the model folder name aka. the value in this case.